src/utilities/fetch-supporters.js

```diff
@@ -42,7 +42,7 @@ const membersGraphqlQuery = `query account($limit: Int, $offset: Int) {
 }`;
 
 // only query transactions in last year
-const transactionsGraphqlQuery = `query transactions($dateFrom: ISODateTime, $limit: Int, $offset: Int) {
+const transactionsGraphqlQuery = `query transactions($dateFrom: DateTime, $limit: Int, $offset: Int) {
```

Comment:
Good job on fixing this.

This was indeed a recent breaking change. We thought it was only used internally and decided to proceed. We should have better monitored usage.